policyengine_us/variables/gov/states/de/tax/income/deductions/de_aged_deduction.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class de_aged_deduction(Variable):
```

Comment:
Make this a single variable `de_additional_standard_deduction` combining aged and blind, reflecting the legal code and form instructions. The `standard` part is important, as it reflects the requirement to not itemize deductions to claim it.
```suggestion
class de_aged_deduction(Variable):
```